### PR TITLE
Update planetusermanual.md Adding Language Overview Issue #590

### PR DESCRIPTION
**Pull request by Loshma93**

### CODE REVIEW

File: pages/planetusermanual.md [hunk 1]
```diff
@@ -294,3 +294,9 @@ Feedback allows Learners to provide feedback on the currently open page. Feedbac
 Learners can give feedback for any page or Resource at any time by click on “Feedback” and filling out the information in the form shown below.  When finished, click “Submit” to send feedback to OLE.
 
 ![](uploads/images/planet_feedback.png)
+
+## Language Overview
+### _How do I change the Language?_
+
+Learners can change the language of the PLANET system at any time by going to the Home page and selecting the language pull down menu  from in the top right corner.
+![](uploads/images/Planet_Language.png)
```
Comment (Yurockkk) [inline pages/planetusermanual.md]: @Loshma93 you might want to add an empty line between the paragraph and the image. Right now it seems that the image is on the right side of the paragraph.

**Comment by Loshma93** [comment]
@Yurockkk I added the image to the PR as well. I didn't understand why the image is broken. Can you please let me know why it was broken??

**Comment by Yurockkk** [comment]
@Loshma93 As @mappuji mentioned, right now the image is in the wrong place. Move the image under the directory pages/uploads/images/ might solve the problem 😄

**Comment by Loshma93** [comment]
@Yurockkk Thank you for your suggestion. I will wait suggestions from others and continue with doing accordingly.

**Pull request merged**